frontend/src/lib/utils.tsx

```diff
@@ -1209,3 +1219,7 @@ export function findLastIndex<T>(array: Array<T>, predicate: (value: T, index: n
 export function isEllipsisActive(e: HTMLElement | null): boolean {
     return !!e && e.offsetWidth < e.scrollWidth
 }
+
+export function isGroupType(actor: PersonType | GroupActorType): actor is GroupActorType {
+    return (actor as GroupActorType).group_key !== undefined
```

Comment:
We have `type` in the returned payload though - why don't we use that and instead rely on this field?